setup.py

```diff
@@ -11,6 +11,7 @@
     ("Robert Gieseke", "robert.gieseke@pik-potsdam.de"),
     ("Jared Lewis", "jared.lewis@climate-energy-college.org"),
     ("Sven Willner", "sven.willner@pik-potsdam.de"),
+    ("Chris Smith", "c.j.smith1@leeds.ac.uk")
```

Comment:
I added myself as I wrote the FaIR adapter, but depending on your authorship policies feel free to remove.